### ISSUE TITLE
Templates widget not showing template used

### ISSUE THREAD

**Issue by harrispapadopoulos** [pojome/elementor#6271]
Hello
After latest updates, when clicking on a "template" widget added in a page, it doean't show the template used there.
Instead in the settings panel it shows the option to choose a new template (alllthough template is already chosen). 
This makes it difficult to find out what template was used there (espesially in case somebody else has built the template).

This used to work properly before latest updates.
Please take care

Thanks!

**Comment by bainternet** [comment]
@harrispapadopoulos 	

Thanks, Its already fixed and will be release soon.

**Comment by black-eye** [comment]
Yes, I can see name of the template now, but Edit Template button is missing in return:

![obrazek](https://user-images.githubusercontent.com/5634870/49193547-b651a400-f37f-11e8-899e-5870ae6f94ca.png)

(It is visible, while loading the Template name, but then disappears).

I'm also not sure, wheather the search function works as intended - it seems that the text written doesn't have a real impact on filtering the templates:
![obrazek](https://user-images.githubusercontent.com/5634870/49193505-97531200-f37f-11e8-9389-8368da98b588.png)
